.github/workflows/tests.yml

```
@@ -0,0 +1,92 @@
+name: Tests
+on: [push, pull_request]
+
+jobs:
+    tox-py:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python-version: ['3.8.1']
```

This is not safe to do like this - when Python gets updated in Actions's virtual environment, previous micro version is no longer gonna be there causing the whole workflow to error. Same applies to all other occurrences of "3.8.1"
```suggestion
          python-version: ['3.8']
```

---

We can wait a few days until the live release of 3.8.1 for GHA, so I think it would be fine just leaving it like this until its ready.